Form/Type/ColorSelectorType.php

```diff
@@ -48,7 +48,15 @@ public function configureOptions(OptionsResolver $resolver)
                 Colors::WHITE,
                 Colors::YELLOW,
             ),
-        ));
+        );
+
+        // SF >2.8
+        if (method_exists('Symfony\Component\Form\AbstractType', 'getBlockPrefix')) {
+            $defaults['choices'] = array_unique(array_flip(Colors::getAll()));
```

Comment:
This would be a BC-break, and would therefore require a BC-break handling mechanism : 
- introducing an option to make `getAll()` return a flipped array of unique_key
- triggering a deprecation warning when the option is not set
- removing the option on `NEXT_MAJOR`